infra/modules/psoxy-package/main.tf

```diff
@@ -39,5 +31,9 @@ output "path_to_deployment_jar" {
 }
 
 output "filename" {
-  value = local.filename
+  value = data.external.deployment_package.result.filename
+}
+
+output "version" {
```

Comment:
I think `version` can be removed, as far as I can tell is not used and value comes embedded in filename.

Reply:
i think it's good to have the structured data there if we ever need it,